proposals/0207-rpc-message-protection.md

```diff
@@ -70,7 +70,7 @@ We also add a new result code. SDL sends this code to a mobile app when it recei
 </enum>
 ```
 
-In this proposal, we say a RPC message needs encryption/protection if the app does not have `requireEncryption`=`false` in the `OnPermissionsChange` and the RPC has `requireEncryption`=`true` in the `PermissionItem`. We say a RPC message does not need encryption/protection if the app has `requireEncryption`=`false` in the `OnPermissionsChange` or the app does not have `requireEncryption`=`false` in the `OnPermissionsChange` but the RPC has `requireEncryption`=`false` or the `requireEncryption` does not exist in the `PermissionItem`. 
+In this proposal, RPC messages need encryption/protection if the app does not have `requireEncryption`=`false` in the `OnPermissionsChange` and the RPC has `requireEncryption`=`true` in the `PermissionItem`. The RPC message does not need encryption/protection if the app has `requireEncryption`=`false` in the `OnPermissionsChange` or the app does not have `requireEncryption`=`false` in the `OnPermissionsChange` and either the RPC has `requireEncryption`=`false` or the `requireEncryption` does not exist in the `PermissionItem`. 
```

Comment:
Please update to this format as it is very hard to follow at its current state. 

In this proposal, RPC messages need encryption/protection if the app does not have `requireEncryption`=`false` in the `OnPermissionsChange` and the RPC has `requireEncryption`=`true` in the `PermissionItem`. 
The RPC message does not need encryption/protection if:
-  The app has`requireEncryption`=`false` in the `OnPermissionsChange`.
- The app does not have `requireEncryption`=`false` in the `OnPermissionsChange`.
- Either **of** the **RPCs** **have** `requireEncryption`=`false`.
- The`requireEncryption` does not exist in the `PermissionItem`.

Reply:
In this proposal, RPC messages need encryption/protection if the app does not have `requireEncryption`=`false` in the `OnPermissionsChange` and the RPC has `requireEncryption`=`true` in the `PermissionItem`. 

The RPC message does not need encryption/protection if:
- The app has `requireEncryption`=`false` in the `OnPermissionsChange`. 
- The app does not have `requireEncryption`=`false` in the `OnPermissionsChange` and either the RPC has `requireEncryption`=`false` or the `requireEncryption` does not exist in the `PermissionItem`.